### ISSUE TITLE
Chart for battery temperature

### ISSUE THREAD

**Issue by thueske** [tillsteinbach/VWsFriend#594]
hello,

With the VW ID integration for homeassistant, it is now possible to display a sensor for the temperature of the HV battery.

https://github.com/mitch-dc/volkswagen_we_connect_id/pull/224/files

Is it also possible to have a chart for this in Grafana?

**Comment by tillsteinbach** [comment]
Yes, this will come soon. With the latest version the data is already being collected in the database.

**Comment by claesto** [comment]
@thueske you can already add it if you want. I have this for my battery temperature graph:

```
SELECT
    "carCapturedTimestamp",
    "temperatureHvBatteryMin_K",
    "temperatureHvBatteryMax_K"
FROM
    battery_temperature
WHERE
    vehicle_vin = '$VIN'
ORDER BY
    "carCapturedTimestamp" DESC
```

Or use the following code if you'd like the temperatures to be in degrees Celsius instead of Kelvin.

```
SELECT
    "carCapturedTimestamp",
    ("temperatureHvBatteryMin_K" - 273.15) AS "temperatureHvBatteryMin_C",
    ("temperatureHvBatteryMax_K" - 273.15) AS "temperatureHvBatteryMax_C"
FROM
    battery_temperature
WHERE
    vehicle_vin = '$VIN'
ORDER BY
    "carCapturedTimestamp" DESC
```

Of course, yours will be overwritten once the dashboard is updated to include it from the source, but you can make a copy of it so it remains there (or you can use the new one once it's added of course)

**Comment by tillsteinbach** [comment]
Should be fixed in 0.24.5
